docs/manual/lint.md

```
@@ -262,6 +264,116 @@ bunx eslint .
+    "@fedify/lint/actor-id-mismatch": "error",
+    "@fedify/lint/actor-inbox-property-required": "warn",
+    "@fedify/lint/actor-outbox-property-required": "warn",
+    "@fedify/lint/actor-followers-property-required": "warn",
+    "@fedify/lint/actor-public-key-required": "warn",
+    "@fedify/lint/actor-assertion-method-required": "warn",
+    "@fedify/lint/collection-filtering-not-implemented": "warn"
+  }
+}
+~~~~
+
+[Rules]: #rules
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "rules"

(MD053, link-image-reference-definitions)